### PR TITLE
Publicize some functions in PreviewingController file

### PR DESCRIPTION
**Pull request by kimmanuel**
- Make some functions available to other files to be able to use the peekpop view in MKAnnotations since annotations cannot detect long tap gestures using BAPeekPop
- Changed text color of actions buttons

### CODE REVIEW

File: BAPeekPop/Classes/BAPreviewingViewController.m [hunk 1]
```diff
@@ -92,7 +92,7 @@ - (void)viewDidLoad {
         
         // 2. add arrow-up icon
         self.arrowUpImageView = ({
-            UIImage *image = [UIImage imageNamed:@"icon-arrow-up" inBundle:[NSBundle bundleForClass:[BAPreviewingViewController class]] compatibleWithTraitCollection:nil];
+            UIImage *image = [UIImage imageNamed:@"icon_peek_pop_arrow_up"];
```
Comment (BramYeh) [inline BAPeekPop/Classes/BAPreviewingViewController.m]: @kimmanuel 
Would you please upload icon_peek_pop_arrow_up.png?

File: Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (BramYeh) [inline Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata]: maybe we can ignore this file?

**Review by BramYeh — COMMENTED** [review]
It's tricky that CI reported 
BAPeekPop.h:11:9: 'BAPreviewingViewController.h' file not found 

I would trigger again to investigate what was happened.

File: BAPeekPop/Classes/BAPreviewingViewController.m (hunk 1 above)
Comment (kimmanuel) [reply BAPeekPop/Classes/BAPreviewingViewController.m]: Hi @BramYeh, I realized it's the same image as "icon-arrow-up", so I reverted the code back.

**Comment by kimmanuel** [comment]
Hi @BramYeh, this PR is ready for Code Review now. Thanks for looking on this!

**Comment by BramYeh** [comment]
Hi, @kimmanuel 
I found Travis cannot build successfully because BAPreviewingViewController.h isn't public header.
I would merge this PR first and correct it.
Thanks for your assistance to refine color and make it supports MKAnnotations

**Pull request merged**